flang/lib/Lower/OpenMP.cpp

```diff
@@ -112,7 +112,8 @@ static void gatherFuncAndVarSyms(
 
 static Fortran::lower::pft::Evaluation *
 getCollapsedLoopEval(Fortran::lower::pft::Evaluation &eval, int collapseValue) {
-  // Return the Evaluation of the innermost collapsed loop.
+  // Return the Evaluation of the innermost collapsed loop, or the current one
+  // if there was no COLLAPSE.
   if (collapseValue == 0)
```

Comment:
I actually missed this change in the previous commit, but turns out that it's actually useful in this PR, since this function is called separately in a couple of places.